hack/test_errata.py

```diff
@@ -0,0 +1,738 @@
+import datetime
+import errata
+import github
```

Comment:
nit: this is not a stdlib dependency, so it should go [in it's own block after the stdlib stuff][1].

[1]: https://www.python.org/dev/peps/pep-0008/#imports